voters-guide-2017.md

```diff
@@ -0,0 +1,69 @@
+
+## PROCESS
+Elections will be held using time-limited [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on [CIVS](http://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method). The top vote getters, modulo the corporate diversity requirement of no more than 1/3 of the committee from a single company, will be elected to the respective positions, with the top 3 filling the 2 year term seats, and the next 3 filling the 1 year term seats. 
+
+The election will open for voting on September 19, 2017 at 09:00am PDT and end two weeks after on October 3, 2017 at 06:00pm PDT. You will receive an email to the address on file at the start of the election from <civs@cs.cornell.edu>; please whitelist if necessary. 
+
+## ELIGIBILITY 
+Members of Standing are defined by the union of:
+SIG leads
+Approvers and reviewers in any Kubernetes owned repositories
+Anyone with write access to a Kubernetes owned repository
+Active members of our community
+If you believe you are a Member in Standing, please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSeoeNSl9ufZ_jpp7OHgvtWm-GRFV6WUwTIqZ9W25eMd3xyyvg/viewform) <strong><em>before</strong></em> September 13, 2017.	
+
+## DECISION
+The newly elected body will be announced in the weekly Kubernetes Community Meeting on October 5, 2017 at 10:00am US Pacific Time. [Please join us](https://groups.google.com/forum/#!forum/kubernetes-community-video-chat).
+
+Immediately following the meeting, the raw voting results and winners will be published on the [Kubernetes Blog](http://blog.kubernetes.io/).
+
+For more information, definitions, and/or detailed election process, see full [steering committee charter](https://github.com/kubernetes/steering/blob/master/charter.md).
+
+# NOMINEES
+Don’t know someone on this list? We asked the nominees to provide a <300 word bio about their k8s experience that we'll link to their name in this table. If you don't see a link, try [here](https://github.com/kubernetes/community/tree/master/community/elections/2017) under first+lastname.  
+
+Name | Organization/Company | GitHub 
+--- | --- | --- 
+Aaron Crickenberger	| Samsung SDS |	[@spiffxp](https://github.com/spiffxp)
+Aaron Schlesinger |	Microsoft |	[@arschles](https://github.com/arschles)
+Adnan Abdulhussein |	Bitnami |	[@prydonius](https://github.com/prydonius)
+Alex Pollitt |	Tigera	| [@lxpollitt](https://github.com/lxpollitt)
+Caleb Miles | CoreOS |	[@calebamiles](https://github.com/calebamiles)
+Derek Carr |	Red Hat |	[@derekwaynecarr](https://github.com/derekwaynecarr)
+Doug Davis |	IBM	| [@duglin](https://github.com/duglin)
+Ihor Dvoretskyi |	Mirantis |	[@idvoretskyi](https://github.com/idvoretskyi)
```

Comment:
I don't work for Mirantis anymore. Please, remove this affiliation from the list.